templates/fonts/swift4.stencil

```diff
@@ -64,8 +63,13 @@
     {% for font in family.fonts %}
     {{accessModifier}} static let {{font.style|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}} = FontConvertible(name: "{{font.name}}", family: "{{family.name}}", path: "{% call transformPath font.path %}")
     {% endfor %}
+    {{accessModifier}} static let all: [FontConvertible] = [{% for font in family.fonts %}{{font.style|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}}{% if not forloop.last %}, {% endif %}{% endfor %}]
```

Comment:
I'd make these multiline? It's what we do in (for example) the assets templates. Same comment for the other `all` constant.

Reply:
I wondered about that. But I figured we rarely have hundreds of fonts in a given font family, at most 5-10, and we rarely have hundreds of custom font families either (unlike assets), so…

Reply:
True, let's keep it as a oneliner.